.storybook/helpers/files.data.ts

```diff
@@ -50,15 +70,26 @@ export const filesRoot: File[] = [
     ],
   },
   {
+    disabled: true,
     name: 'package.json',
   },
   {
     name: 'tsconfig.json',
   },
 ];
 
-export function getFileTreeNodeMarkup(files: File[] = filesRoot) {
+export function getFileTreeNodeMarkup(files: File[] = filesRoot, args = {}) {
```

Comment:
```suggestion
export function getFileTreeNodeMarkup(files: File[] = filesRoot, args: { asLink?: boolean, clrSelected?: ClrSelectedState } = {}) {
```

Missing type for `args`.

Reply:
corrected in commit e22dd2e